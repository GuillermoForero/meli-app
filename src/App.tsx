import AppProvider from 'providers/AppProviders';
import React from 'react';
import Router from 'router';

const App = () => (
  <AppProvider>
    <Router />
  </AppProvider>
);

export default App;
