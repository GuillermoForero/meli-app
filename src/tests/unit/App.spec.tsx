import React from 'react';
import App from 'App';
import { render } from 'tests/support/customRender';

describe('[App]', () => {
  it('Renders App', () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
