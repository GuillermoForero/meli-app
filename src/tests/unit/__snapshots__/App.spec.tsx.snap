// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[App] Renders App 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <p>
        hola
      </p>
    </header>
  </div>
</div>
`;
